pronto_mia/lib/core/models/error_analyzer.dart

```diff
@@ -0,0 +1,67 @@
+import 'package:graphql/client.dart';
+
+class ErrorAnalyzer {

```

Comment:
I think this doesn't belong to "models", maybe the factories folder is a better place.